js/src/forum/components/DiscussionPage.js

```diff
@@ -88,6 +89,24 @@ export default class DiscussionPage extends Page {
   view() {
     const discussion = this.discussion;
 
+    let content;
+
+    if (discussion) {
+      const stream = new PostStream({ state: this.streamState });
+      stream.on('positionChanged', this.positionChanged.bind(this));
```

Comment:
Won't this add the event handler oh every redraw?

Reply:
It will, I wasn't sure of a better way to do this. Ideally, that event would not be coming from the component class at all, but I wasn't sure of the best way to decomponentize it.

Reply:
No longer relevant as it's passed via a prop now.